JASP-Engine/JASP/R/commonMachineLearningClassification.R

```diff
@@ -873,7 +873,7 @@
     typeData <- cbind(train, levelVar = factor(levelVar))
     typeData <- typeData[, -which(colnames(typeData) == .v(options[["target"]]))]
     
-    score <- .calcAUCScore(AUCformula, test, typeData, levelVar, options, ...)
+    score <- .calcAUCScore(AUCformula, train, test, typeData, levelVar, options, ...)
```

Comment:
If you change this to
`score <- .calcAUCScore(AUCformula, train=train, test=test, typeData=typeData, levelVar=levelVar, options=options, ...)`
Then you don't have to specify `train` in the S3Methods that don't actually use that parameter (=`ldaClassification`, `boostingClassification`, `knnClassification`)

Will make it a little easier to make changes in the future.

Reply:
yes, done